README.md

```diff
@@ -1 +1,28 @@
 # DR2 End to end tests
+
+These are tests of the DR2 ingest process. They only include tests cases that will be successfully ingested into
+Preservica.
+Test cases which will fail mid-process will be dealt with in unit tests on the relevant lambdas.
```

Comment:
Missing comma after "mid-process"

Reply:
I think it's also right without it but I've put one in.